local_CI.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/env sh
```

Comment:
btw, this hashbang makes all the scripts invalid for execution on quite a lot of systems - Mac for example puts `env` in `/usr/bin` with no symlink of it from `/bin`.

Personally I actually thought `env` typically lived in `/usr/bin` on most systems, but I guess not if it's in `/bin` on yours.

Reply:
Oh yeah. I have `bin ⇒ usr/bin`